.goreleaser.yml

```diff
@@ -30,6 +30,12 @@ builds:
 archives:
 - format: zip
   name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
+sboms:
+- documents:
+  - bom.json
+  artifacts: any
+  cmd: cyclonedx-gomod
+  args: [ "mod", "-assert-licenses", "-std", "-json", "-verbose", "-output", "$document", "../" ]
```

Comment:
This will need to be modified just like the others as it won't have all the information required in the generated SBOM.

Reply:
Made it so it generates an SBOM for each binary. IIUC, it should have the right options set. I don't see a way we could merge the different SBOMs, however.

Reply:
Tried testing it by pushing up a tag but it doesn't seem like GitHub will let me create a test release without everything being merged in `main`.

Reply:
We have another issue that is to flush out the sbom details.